packages/console/src/ConsoleInput.jsx

```diff
@@ -7,6 +7,7 @@ import Log from '@deephaven/log';
 import { PromiseUtils } from '@deephaven/utils';
 import { getCommandHistoryStorage } from '@deephaven/redux';
 import MonacoUtils from './monaco/MonacoUtils';
+import MonacoCompletionProvider from './MonacoCompletionProvider';
```

Comment:
It would make sense to move this into `console/src/monaco` as well, then make an index file that exports all of them, so you can just do `import { MonacoCompletionProvider, MonacoTheme, MonacoUtils } from './monaco'`, would be nice little cleanup.

Reply:
Does that work for MonacoTheme, even though it is imported `from ./monaco/MonacoTheme.module.scss` ?